cartographer/cloud/internal/mapping/serialization.cc

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2018 The Cartographer Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "cartographer/cloud/internal/mapping/serialization.h"
+
+namespace cartographer {
+namespace cloud {
+
+proto::TrajectoryState ToProto(
+    const mapping::PoseGraphInterface::TrajectoryState& trajectory_state) {
+  using TrajectoryState = mapping::PoseGraphInterface::TrajectoryState;
```

Comment:
ah, just a nit: you can move this `using ...` outside of both functions into `namespace{}` and specify the type completely,i.e. `::cartographer::...`.